core/unix/coredump.c

```diff
@@ -45,6 +45,9 @@
 #include "dr_tools.h"
```

Comment:
> i7046: Add NT_ARM_TLS note to store tpidr_el0 for Aarch64

style: missing `#`: `s/i7046/i#7046/`

Also, please see https://dynamorio.org/page_code_reviews.html#sec_commit_messages: for multiple commits to the same issue, there should be a very short description.  So it should be `i#7046 memory dump: Add...` or something like that.